packages/core/src/divider/divider.stories.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@clr/core/button/register.js';
+import '@clr/core/divider/register.js';
+import { getElementStorybookArgTypes, spreadProps, getElementStorybookArgs } from '@clr/core/internal';
+import { html } from 'lit-html';
+import customElements from '../../dist/core/custom-elements.json';
+
+export default {
+  title: 'Components (Preview)/Divider/Stories',
+  component: 'cds-divider',
+  argTypes: getElementStorybookArgTypes('cds-divider', customElements),
+  parameters: {
+    options: { showPanel: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=51%3A673',
+    },
+  },
+};
+
+export const API = (args: any) => {
+  return html`
+    <cds-divider ...="${spreadProps(getElementStorybookArgs(args))}">
+      ${args.default}
+    </cds-divider>
+  `;
+};
+
+export const horizontal = () => {
+  return html`
+    <div cds-layout="vertical gap:md">
+      <p>It is often preferable to use dividers inside of layouts.</p>
+      <cds-divider></cds-divider>
+      <p>This allows them to span the dimensions of their containers as expected.</p>
```

Comment:
Can we update all the `<p>` tags in the divider stories to have a `cds-text="body"` attribute?

Reply:
Yup. I can do that.

Reply:
Done.